tests/functional/pv/pv_services/test_cephfs_fsync_consistency.py

```
@@ -52,24 +52,26 @@ def write_and_read(self, client_pod_obj, server_pod_obj):
                 cmd=f"oc cp {local_script_path} {client_pod_obj.name}:/tmp/sync_script.py"
             )
         except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)